services/pull/pull.go

```diff
@@ -838,3 +866,158 @@ func IsHeadEqualWithBranch(ctx context.Context, pr *issues_model.PullRequest, br
 	}
 	return baseCommit.HasPreviousCommit(headCommit.ID)
 }
+
+// GetCodeOwners returns the code owners configuration
+// Return empty slice if files missing
+// Return error on file system errors
+// We're trying to do the best we can when parsing a file.
+// Invalid lines are skipped. Non-existent users and groups too.
+func GetCodeOwners(ctx context.Context, repo *repo_model.Repository, branch string) ([]*CodeOwnerRule, error) {
+	files := []string{"CODEOWNERS", "docs/CODEOWNERS", ".gitea/CODEOWNERS"}
```

Comment:
Thanks for this PR <3 Love the comments, and that docs are included too.

Could you add `.github/CODEOWNERS` here too to ensure compatibility when migrating from GH to Gitea.

Reply:
That's reasonable. Thanks for clarification :)